packages/main/test/pages/Dialog.html

```diff
@@ -292,19 +292,19 @@
 	</ui5-dialog>
 
 
-	<ui5-dialog id="prevent-dialog" header-text="Message dialog">
-		<p>Prevent closing the dialog.</p>
-
-		<div slot="footer" class="dialogFooter">
-			<ui5-button design="Emphasized">OK</ui5-button>
+	<ui5-dialog id="prevent-dialog" header-text="Fiori footer buttons">
+		<p>Adding custom styles to achieve the look and field of a Fiori footer</p>
```

Comment:
I suggest:
Adding custom styles to achieve the look and feel of a SAP Fiori footer